src/Inline/Element/AbstractInlineContainer.php

```diff
@@ -19,14 +19,14 @@
     /**
      * @var AbstractInline[]
      */
-    protected $children = [];
+    private $children = [];
```

Comment:
What's the rationale for making children `private` and having parents be `protected`?  Should they both be `private` or `protected`?  (I haven't thought through this - just throwing the question out there)